python/rmm/_lib/device_buffer.pxd

```diff
@@ -51,6 +47,7 @@ cdef class DeviceBuffer:
     @staticmethod
     cdef DeviceBuffer c_frombytes(const unsigned char[::1] b,
                                   uintptr_t stream=*)
+    cpdef copy_to_host(self, unsigned char[::1] hb=*, uintptr_t stream=*)
```

Comment:
> > Also adds a method of the same name to DeviceBuffer to match what Numba does with DeviceNDArrays.
> 
> Is that still not added or did I miss something?

Here's the definition. The implementation is below in the `.pyx` file 🙂

Reply:
Ah, got it now. I misinterpreted your sentence, sorry about that.

Reply:
Was there something else you were hoping to see here?

Reply:
Nope, I just thought you were saying that a new method _with the name_ `DeviceBuffer` was added, but you said you added a method of _the same name to_ `DeviceBuffer` (where same name is `copy_to_host`). My mistake.